### ISSUE TITLE
change "is binary" file property to "filetype", to distinguish between: text, image, executeable

### ISSUE THREAD

**Issue by y0014984** [y0014984/Advanced-Equipment#175]

**Comment by GermanHydrogen** [comment]
How would you implement the "image" filetype?
My idea with "is binary" was, that if the file is not binary, a secondary programm would handle filetypes.

**Comment by GermanHydrogen** [comment]
I agree with you, that for the interface a distinction would be usefull. Also, depending on the implementation, some filetypes
would need to be preprocessed before usage in the terminal.
I don't think, we need to make the distinction in the filesystem if the terminal programs just fail, if the incorrect filetype was 
used as an input.
